src/Service/Migrations/Mssql/20240108104909_AddTablePeriodicCredentialReports.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Passwordless.Service.Migrations.Mssql;
+
+/// <inheritdoc />
+public partial class AddTablePeriodicCredentialReports : Migration
+{
+    /// <inheritdoc />
+    protected override void Up(MigrationBuilder migrationBuilder)
+    {
+        migrationBuilder.CreateTable(
+            name: "PeriodicCredentialReports",
+            columns: table => new
+            {
+                Tenant = table.Column<string>(type: "nvarchar(450)", nullable: false),
+                CreatedAt = table.Column<DateTime>(type: "datetime2", nullable: false),
```

Comment:
@jonashendrickx Did you re-generate these after adding DateOnly? I would expect this to be date in sqlserver 👀